README.md

```diff
@@ -215,6 +217,24 @@ hints and first-parties can use `Feature-Policy` to decide which third-parties t
 privilege with detailed user agent information. In any event, moving to this opt-in model means
 that the extent of a site's usage can be monitored and evaluated.
 
+For developers that prefer using user agent information to make client-side decisions, they could use the JavaScript API:
+
+```javascript
+  const uaData = navigator.userAgentData;
```

Comment:
I've seen this pattern (immediately assigning the entire thing to a variable and then pulling info out of the variable) a couple of times. For my own edification, is this because "navigator.userAgentData" runs an expensive operation each time it's accessed, correct?

---

A couple of follow up random thought, are these values frozen to the document, or could they change? For instance, if I had a button that called these attributes, and the prefs changed in the background between button presses, would these values change? 

Also also, can values change between calling `userAgentData` and `getHighEntropyValues` returning? Like, could I get `{brand: "Google Chrome", version: "84"}` and `85.2.345.2` on the same page?

I can spin these off into their own issue if it's too much to answer here